i18n/zh/docusaurus-plugin-content-docs/current/userguide/scheduling/resource-propagating.md

```diff
@@ -671,3 +671,74 @@ spec:
 ```
 The `nginx` deployment in `default` namespace will be propagated to cluster `member2`.
 
+
+## 资源分发的暂停与恢复
```

Comment:
```suggestion
## 暂停与恢复资源分发
```
Heandings can have a consisten style of `v+n`

Reply:
Thanks for the clear instructions.